collection/customsitescollection.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ *
+ * phpBB Media Embed PlugIn extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2019 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\mediaembed\collection;
+
+/**
+ * Collection of custom site definitions
+ *
+ * See https://s9etextformatter.readthedocs.io/Plugins/MediaEmbed/Add_custom/
+ */
+class CustomSitesCollection
+{
+	/** @var array $custom_sites_collection Custom site definitions */
+	protected $custom_sites_collection = [
+		'ok' =>
```

Comment:
'ok.ru'

Reply:
Well, I think the site ID is intended to be used as a BBCode ```[ok][/ok]```, so ```ok.ru``` wouldn't be good here.

Reply:
Oh you're right. That just seemed so small and generic.